src/clients/UBAClient/UBAClientTypes.ts

```diff
@@ -50,3 +50,19 @@ export type UBABundleState = {
 export type UBAClientState = {
   [chainId: number]: UBAChainState;
 };
+
+export type SpokePoolEventFilter = {
```

Comment:
Not sure I agree with adding these back until we need them. For example, I don't see any of the relayer bots using the `originChainId` or `destinationChainId` filters currently